Contents/Code/PAsearchSites.py

```diff
@@ -849,6 +850,7 @@
 searchSites[760] = ["Clips4Sale","Clips4Sale","https://www.clips4sale.com","https://www.clips4sale.com/studio/"]
 searchSites[761] = ["VogoV","VogoV","https://vogov.com","https://vogov.com/search/?q="]
 searchSites[762] = ["Ultrafilms","Ultrafilms","https://www.ultrafilms.xxx","https://www.ultrafilms.xxx/?s="]
+searchSites[763] = ["Fuckingawesome","Fuckingawesome","https://fuckingawesome.com","https://fuckingawesome.com/search/videos/"]
```

Comment:
May want to update these to "FuckingAwesome" (vs "Fuckingawesome") so that capitalization is the same for the studio, collection, and tagline.

Reply:
done